packages/perseus-editor/src/components/sortable.tsx

```diff
@@ -134,40 +163,45 @@ const SortableArea = createReactClass({
                 {sortables}
             </ol>
         );
-    },
-});
+    }
+}
+
+type ItemProps = {
+    area: any;
+    component: React.ReactNode;
+    dragging: boolean;
+    draggable: boolean;
+    index: number;
+};
 
 // An individual sortable item
-const SortableItem = createReactClass({
-    propTypes: {
-        area: PT.shape({
-            onDragEnter: PT.func.isRequired,
-            onDragStart: PT.func.isRequired,
-            onDrop: PT.func.isRequired,
-        }),
-        component: PT.node.isRequired,
-        dragging: PT.bool.isRequired,
-        draggable: PT.bool.isRequired,
-        index: PT.number.isRequired,
-    },
-    handleDragStart: function (e) {
+export class SortableItem extends React.Component<ItemProps> {
```

Comment:
This class is the `default` export so we don't need to export this as a named export also.